lib/Service/CallService.php

```diff
@@ -169,4 +169,34 @@ public function call(
 
 		return $callLog;
 	}
+
+	/**
+	 * Uses Adbar Dot to place the values of keys with a dot in it in the $config array
+	 * to the correct position in the then updated multidimensional $config array.
+	 *
+	 * @param array $config The config array.
+	 *
+	 * @return array The updated config array.
+	 */
+	private function applyConfigDot(array $config): array
+	{
+		$dotConfig = new Dot($config);
+		$unsetKeys = [];
+
+		// Check if there are keys containing a dot we want to map to a different position in the $config array.
+		foreach ($config as $key => $value) {
+			if (str_contains($key, '.')) {
+				$dotConfig->set($key, $value);
+				$unsetKeys[] = $key;
+			}
+		}
+
+		// Remove the old keys containing a dot that we mapped to a different position in the $config array.
+		$config = $dotConfig->all();
+		foreach ($unsetKeys as $key) {
+			unset($config[$key]);
+		}
```

Comment:
To remove these:
![image](https://github.com/user-attachments/assets/e4dc669c-0f0b-4de0-ab0f-c83457bba59f)

Otherwise they would still be present here:
![image](https://github.com/user-attachments/assets/3829ca15-5fcc-4e44-8e81-d359a3f6ad6d)